sysconfigs/daint/modules.yaml

```diff
@@ -76,6 +76,11 @@ modules:
         set:
           UCX_MEMTYPE_CACHE: 'n'
           UCX_TLS: 'rc_x,ud_x,mm,shm,cma'
+    cosmoc2sm cosmo_target=cpu:
+      environment:
+        set:
+          UCX_MEMTYPE_CACHE: 'n'
+          UCX_TLS: 'rc_x,ud_x,mm,shm,cma'
```

Comment:
This should be set automatically by the openmpi dependency, did you add those lines because it didnt work?